autopkg/workflows/autopkg.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     timeout-minutes: 90 # Keeps your builds from running too long
     steps:
     - name: Checkout AutoPkg recipes
-      uses: actions/checkout@01aecccf739ca6ff86c0539fbc67a7a5007bbc81 # Pin SHA1 hash instead of version
+      uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b # Pin SHA1 hash instead of version
```

Comment:
Should we be updating the sha1 hashes in this commit?

Reply:
This is to prevent drift from our internal runs and the public repo.